### PR TITLE
Add show/hide all selections for PDF

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

**Review by sdreher — COMMENTED** [review]
Reviewing this in my dev env...I don't see this working in the selection assignment. Is it hooked up yet?

**Comment by nikolas** [comment]
Most of the issues here have been resolved from my perspective - but still one lingering problem: when the window is resized all but one of the selections disappear.

**Comment by sdreher** [comment]
@nikolas - lmk if you're ready for a review, or if you want to try to get the resize working.

**Comment by nikolas** [comment]
alright, this is finally ready for another review @sdreher. Some more changes were required: saving the annotations in a state object to re-display them as the page is re-rendered on resize.

File: media/js/app/assetmgr/asset.js [hunk 1]
```diff
@@ -733,6 +749,11 @@
         };
 
         this.showAnnotation = function(event, ui) {
+            if (document.forms['annotation-list-filter']) {
+                document.forms['annotation-list-filter']
+                    .elements.showall.checked = false;
```
Comment (sdreher) [inline media/js/app/assetmgr/asset.js:754]: I'm not sure about setting the `showall` to false here. Currently, the image viewer continues to show all selections, and simply focuses on the selected one. (Image also has different colored borders, but we can leave that for another time.)

**Review by sdreher — COMMENTED** [review]
Show all is working nicely. :tada: Please take a look at the question on setting showall to false when a single selection is clicked.

File: media/js/app/assetmgr/asset.js (hunk 1 above)
Comment (nikolas) [reply media/js/app/assetmgr/asset.js]: Alright, the showall behavior should be closer to how it works on the Image type, now.

**Pull request merged**